de/manual/ch_Customize.docbook

```diff
@@ -3136,29 +3136,29 @@ Scrolling to older transactions is not possible. If you want to see all entries
 
                   <listitem>
                     <para>Verwenden Sie die <guibutton>Durchsuchen</guibutton>-Schaltfläche, um ein Bild auszuwählen, das
-                      als Hintergrund in den Berichten verwendet werden soll. Mit der Schaltfläche
+                      als Hintergrund in den Berichten platziert werden soll. Mit der Schaltfläche
                       <guibutton>Zurücksetzen</guibutton> können Sie die Auswahl wieder löschen.
```

Comment:
`<guibutton>Zurücksetzen</guibutton>` scheinen wir öfter zu brauchen, sollte also in die dtd.